EnglishFilter/sections/adservers.txt

```diff
@@ -8,6 +8,12 @@
 ||voawhooh.net^$third-party
 ||suptaibo.com^$third-party
 ||topcpmnetwork.com^$third-party
+||ypofozux.xyz^
+||chfpgcbe.com^$third-party
+||crevicedepressingpumpkin.com^
+||peuzubuqo.com^$third-party
+||viqirue.ru^
+||matchaix.net^$third-party
```

Comment:
Some of your rule is already merged in your previous pull request merged
https://github.com/AdguardTeam/AdguardFilters/pull/75224/files/35efee582939d43b0ae50c91528c413ea45fa59a

Reply:
Fixed.

Reply:
This entry eg
`||matchaix.net^$third-party` is already included in adguard

Reply:
I didn't even add it, in this pull request.